server/pkg/integrationapi/schema.go

```diff
@@ -56,3 +73,74 @@ func NewSchema(i *schema.Schema) Schema {
 		CreatedAt: lo.ToPtr(i.ID().Timestamp()),
 	}
 }
+
+func NewFieldChangeType(value string) *FieldChangeType {
+	ft := FieldChangeType(value)
+	return &ft
+}
+
+func NewItemChange(n []*item.Field, o []*item.Field) []FieldChange {
```

Comment:
This logic should be put in item package. Also define FieldChange struct in item package. Comparing items to each other to get the difference is a fairly generic process that you may want to use in other situations in the future.